src/features/import/hooks/useConfigure.ts

```diff
@@ -36,44 +63,46 @@ export default function useConfigure(orgId: number) {
       dispatch(
         importPreviewAdd({
           problems: [],
-          stats: {
-            person: {
-              summary: {
-                addedToOrg: {
-                  byOrg: {},
-                  total: 0,
-                },
-                created: {
-                  total: 0,
-                },
-                tagged: {
-                  byTag: {},
-                  total: 0,
-                },
-                updated: {
-                  byChangedField: {},
-                  byInitializedField: {},
-                  total: 0,
-                },
-              },
-            },
-          },
+          stats: emptyStats,
         })
       );
     } else {
-      const importOperations = prepareImportOperations(
-        configuredSheet,
-        countryCode
-      );
-
-      const previewRes = await apiClient.post<
-        ImportPreview,
-        ZetkinPersonImportPostBody
-      >(`/api/orgs/${orgId}/bulk/preview`, {
-        ops: importOperations,
-      });
+      let importOperations;
+      try {
+        importOperations = prepareImportOperations(
+          configuredSheet,
+          countryCode
+        );
+      } catch (error) {
+        return addUnexpectedErrorToProblems();
```

Comment:
To me, the way you have done this is confusing, because it implies to me that `addUnexpectedErrorToProblems()` is a function that returns _something_ and that we want to pass that _something_ on by returning it again from here.

But in reality. `addUnexpectedErrorToProblems()` is a void function, it returns nothing. And that's also true for this (unnamed) function, where this code exists. So I think it would be better to explicitly return nothing, and to not do anything with the (nonexistent, empty, undefined) return value from `addUnexpectedErrorToProblems()`. Like so:

```suggestion
        addUnexpectedErrorToProblems();
        return;
```